examples/getAnnouncements/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ERC5564_CONTRACT,
+  VALID_SCHEME_ID,
+  createStealthClient,
+  getAnnouncements,
+} from 'stealth-address-sdk';
+
+// Example parameters
+const chainId = 11155111; // Example chain ID for Sepolia
+const rpcUrl = 'https://sepolia.infura.io/v3/YOUR_INFURA_PROJECT_ID';
```

Comment:
Let's include a .env.example file for the RPC url.

Reply:
good call

Reply:
[a5b7541](https://github.com/ScopeLift/stealth-address-sdk/pull/15/commits/a5b7541b0be7de96edf9735392166356ce09eeda)

Reply:
updated!

Reply:
[fd49c40](https://github.com/ScopeLift/stealth-address-sdk/pull/15/commits/fd49c40a2eb2e029e577c0b7204f6044deedd7df)